scripts/github-actions/tests-e2e-samples-common

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Common variables for e2e sample tests
+
+# Define the sample regex that will be used in both scripts
+export SAMPLE_REGEX="apigee|backupdr|bigquery|compute|container|monitoring|vertexai|vmwareengine|sql|redis|secretmanager|iam|workstation|connection|feature"
```

Comment:
How did you choose this list? Are they the ones take especially long time? 
This approach may be a little bit hard to extend, could we use alphabetical sharding? e.g. 
 `test-e2e-samples-a-d`, `test-e2e-samples-e-l`, `test-e2e-samples-m-r`, `test-e2e-samples-s-z`.